x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_creation/engine_creation.tsx

```diff
@@ -116,6 +125,91 @@ export const EngineCreation: React.FC = () => {
             </EuiFlexItem>
           </EuiFlexGroup>
           <EuiSpacer />
+          <EuiAccordion id="advancedSettingsAccordion" buttonContent="Advanced settings">
+            <EuiSpacer />
+            <EuiFormFieldset
+              legend={{
+                children: (
+                  <EuiTitle size="xs">
+                    <span>
+                      {i18n.translate('xpack.enterpriseSearch.engineCreation.engineTypeLabel', {
+                        defaultMessage: "Select how you'd like to manage the index for this engine",
+                      })}
+                    </span>
+                  </EuiTitle>
+                ),
+              }}
+            >
+              <EuiFlexGroup direction="column" gutterSize="s">
+                <EuiFlexItem>
+                  <EuiCheckableCard
+                    id="checkableCardId__appSearchManaged"
+                    name="engineTypeSelection"
+                    onChange={() => setEngineType('appSearch')}
+                    checked={engineType === 'appSearch'}
+                    label={
+                      <>
+                        <EuiTitle size="xs">
+                          <span>
+                            {i18n.translate(
+                              'xpack.enterpriseSearch.engineCreation.appSearchManagedLabel',
+                              { defaultMessage: 'I want App Search to manage my data' }
+                            )}
+                          </span>
+                        </EuiTitle>
+                        <EuiSpacer size="xs" />
+                        <EuiText size="s" color="subdued">
+                          <p>
+                            {i18n.translate(
+                              'xpack.enterpriseSearch.engineCreation.appSearchManagedDescription',
+                              {
+                                defaultMessage:
+                                  'Choose from a number of methods for adding documents to your engine including API integration and the Web Crawler.',
```

Comment:
Suggestion: "Create an engine and add documents via Web Crawler, API, or JSON file."